Wikipedia/Code/PageHistoryFilterCountsViewController.swift

```diff
@@ -0,0 +1,130 @@
+import UIKit
+
+protocol PageHistoryFilterCountsViewControllerDelegate: AnyObject {
+    func pageHistoryFilterCountsViewControllerDidDetermineIfFilterCountsAreAvailable(areFilterCountsAvailable: Bool)
```

Comment:
Probably could shrink this name down, IMO I don't think the full view controller type name is needed since it's in the protocol name. Maybe something like 
`func didDetermineFilterCountsAvailability(_ available: Bool, viewController: PageHistoryFilterCountsViewController)`

Reply:
That could work. I always refer to https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html#//apple_ref/doc/uid/20001282-1001803-BCIDAIJE for naming (_Start the name by identifying the class of the object that’s sending the message:_) and in the past I found it confusing when delegate methods didn't include the sender in the name